.github/CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 * @bcumming @msimberg @RMeli
 docs/services/firecrest @jpdorsch @ekouts
 docs/software/communication @msimberg
+docs/software/devtools/linaro @jpghpc
```

Comment:
@jgphpc FYI. Ok for you?

Reply:
if you change the typo in my name, yes ;-)

Reply:
```suggestion
docs/software/devtools/linaro @jgphpc
```

Reply:
Fixed